pages/common/brew.md

```diff
@@ -1,6 +1,7 @@
 # brew
 
 > Package manager for macOS and Linux.
+> See also: `brew-cask`.
```

Comment:
I'm not sure if I get the reasoning for this. Why mention a specific subcommand/not-a-subcommand-anymore?

```suggestion
```